src/frontend/js/components/DashboardItem/DashboardItemEnrollment.tsx

```diff
@@ -0,0 +1,109 @@
+import { defineMessages, useIntl } from 'react-intl';
+import { Enrollment } from '../../types/Joanie';
+import { DashboardItem, DEMO_IMAGE_URL } from './index';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+
+const messages = defineMessages({
+  statusClosed: {
+    id: 'components.DashboardItemEnrollment.statusClosed',
+    description: '',
+    defaultMessage: 'CLOSED • Finished on {date}',
+  },
+  statusOpened: {
+    id: 'components.DashboardItemEnrollment.statusOpened',
+    description: '',
+    defaultMessage: 'COURSE OPEN • Started on {date}',
+  },
+  statusNotActive: {
+    id: 'components.DashboardItemEnrollment.statusNotActive',
+    description: '',
+    defaultMessage: 'NOT ACTIVE',
+  },
+  accessCourse: {
+    id: 'components.DashboardItemEnrollment.gotoCourse',
+    description: '',
+    defaultMessage: 'ACCESS COURSE',
+  },
+});
```

Comment:
Do not translate this content as this is static example waiting to have dynamic values.